Source/GUI/panelsview.cpp

```diff
@@ -209,6 +213,7 @@ void PanelsView::paintEvent(QPaintEvent *e)
         auto dx = availableWidth - m_actualWidth;
 
         m_panelPixmap = QPixmap(availableWidth, availableHeight);
+        m_panelPixmap.fill(Qt::black);
```

Comment:
Rather than black can you reuse the value at https://github.com/bavc/qctools/blob/09c94af972aaf195e732447d21c79f6007dbd24f/Source/GUI/Plot.cpp#L448